shared/constants/types/fs.js

```diff
@@ -70,6 +69,10 @@ export const getPathVisibility = (p: Path): Visibility => {
       return null
   }
 }
+export const getPathVisibility = (p: Path): Visibility => {
```

Comment:
Seems like this isn't used anymore?

Reply:
Also I'm curious about the name "visibility". It seems like just TLF type to me but I must be missing something here.